content/projects/gsoc/2024/project-ideas.html.haml

```diff
@@ -0,0 +1,161 @@
+---
+layout: simplepage
+title: "GSoC 2024 Project Ideas"
+year: 2024
+tags:
+- gsoc
+- gsoc2024
+project: gsoc
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+//The individual page's layout is located at /content/_layouts/gsocprojectidea.html.haml
+
+:css
+  .pi_table {
+    width: 100%;
+  }
+  .pi_table_header {
+    text-align: center;
+  }
+  .pi_table_category {
+    width: 100px;
+  }
+  .pi_table_skills {
+    width: 175px;
+  }
+
+%img{:title => "Jenkins GSoC", :src => expand_link("/images/gsoc/jenkins-gsoc-logo_small.png"), :style => "float:right", :width => "128"}
+
+
+%p
+  = succeed '.' do
+    This page aggregates project ideas for Google Summer of Code
+    = page.year
+  = succeed '.' do
+    See more information about this project and applications
+    %a{:href => expand_link("/projects/gsoc/")}
+      on the Jenkins Google Summer of Code page
```

Comment:
```suggestion
    Refer to the Jenkins Google Summer of Code page
    %a{:href => expand_link("/projects/gsoc/")}
         for more information about this project and applications.
```
Suggestion to update `See` to `Refer to` for accessibility/inclusive naming convention.